ME3Explorer/FaceFXAnimSetEditor/FaceFXAnimSetEditorControl.xaml.cs

```diff
@@ -359,68 +377,76 @@ private void animationListBox_Drop(object sender, DragEventArgs e)
             Window.GetWindow(this).RestoreAndBringToFront();
             if (e.Data.GetDataPresent("FaceFXAnim") && e.Data.GetData("FaceFXAnim") is FaceFXAnimDragDropObject d)
             {
-                int group = d.group;
-                Animation a = d.anim;
-                SelectedLine.AnimationNames.Add(FaceFX.Names.FindOrAdd(a.Name));
-                SelectedLine.Points.AddRange(a.points.Select(x => new FaceFXControlPoint
```

Comment:
Adding the animations points to the selected line should not have been deleted, and neither should adding to NumKeys

Reply:
I deleted these lines because the points in the line and NumKeys list get immediately recalculated based on whatever is in Animations once SaveChanges gets called. Is that not just doing the same thing as these deleted lines?

Reply:
Ah. I see, you added the animation to the Animations list instead. That's a much better way of doing it. Nevermind!

You could change this line though:
`SelectedLine.AnimationNames.Add(FaceFX.Names.FindOrAdd(d.anim.Name));`
to
`FaceFX.Names.FindOrAdd(d.anim.Name);`
No need to add it to the Selected line when that's just going to get recalculated in SaveChanges.

Reply:
Cool, I'll make that change. Thanks for the review!